Simplenote/src/main/res/values/strings.xml

```diff
@@ -189,4 +191,6 @@
     <string name="restore">Restore</string>
     <string name="history">History</string>
     <string name="error_history">Couldn\'t retrieve history.</string>
+    <string name="or">or</string>
+    <string name="sign_in_with_wordpress_com">Sign in with WordPress.com</string>
```

Comment:
Should the "i" in "in" be uppercase or lowercase in this case?  I noticed the other "Sign *" text on the screen uses uppercase (e.g. Sign Up, Sign In).

Reply:
I think I like it better in lowercase, but not quite sure which way is proper:

<img width="307" alt="screen shot 2018-05-23 at 11 42 29 am" src="https://user-images.githubusercontent.com/789137/40444539-7cd9a7a4-5e7e-11e8-92d6-e8c8749bcb41.png">